imageio/imageio-metadata/src/main/java/com/twelvemonkeys/imageio/metadata/tiff/TIFF.java

```diff
@@ -171,6 +171,7 @@ Should probably all map to Java long (and fail if high bit is set for the unsign
 
     int TAG_DATE_TIME = 306;
     int TAG_DOCUMENT_NAME = 269;
+    int TAG_TITLE = 42038;
```

Comment:
Ah, Ok. That's why the tag wasn't picked up when I tried parsing EXIF data containing it?

I will see if I can fix this tonight and push a fix once I have verified that I can parse EXIF data containing the tag.

Reply:
No, it should not matter for parsing. The constants here is only because `TAG_TITLE` reads better than magic numbers liks `42038` or `0xA436` if adding such tags in code. 

If the tag isn't there after parsing, it's either a bug, or you are not looking in the right place (ie. in the normal image IFD, and not the EXIF sub-IFD). 😀

I would need to see code and a sample file to verify...